hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/Processor.java

```diff
@@ -42,30 +41,6 @@
  * <p>
  * The processor accepts input from instances of {@link Inbox} and pushes
  * its output to an instance of {@link Outbox}.
- * <p>
- * By default the processor declares itself as "cooperative" ({@link
- * #isCooperative()} returns {@code true}). It will be assigned an outbox
- * of finite capacity which is not emptied until the processor yields back
- * to the execution engine. As soon as the outbox refuses an offered item,
- * the processor should save its current state and return to the caller.
- * It should also limit the amount of time it spends in the invocation of
- * any of its methods because it will participate in a cooperative
- * multithreading scheme, sharing a thread with other processors.
- * <p>
```

Comment:
This part shouldn't have been deleted, only rephrased.

Reply:
It's duplicate. I put everything to `isCooperative()` method. We can maybe refer from class javadoc to that method.